apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -383,23 +387,37 @@ func (f *Facade) provisioningInfo(model Model, tagString string) (*params.Kubern
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+
 	vers, ok := modelConfig.AgentVersion()
 	if !ok {
 		return nil, errors.NewNotValid(nil,
 			fmt.Sprintf("agent version is missing in model config %q", modelConfig.Name()),
 		)
 	}
-	registryPath, err := podcfg.GetJujuOCIImagePath(controllerCfg, vers)
+
+	modelImage, err := f.broker.GetModelOperatorDeploymentImage()
+	if err != nil {
+		return nil, errors.Annotatef(err, "getting model operator deployment image")
+	}
+
+	modelImageRepo, err := podcfg.RecoverRepoFromOperatorPath(modelImage)
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
 
-	imageRepoDetails, err := docker.NewImageRepoDetails(controllerCfg.CAASImageRepo())
+	modelImagePath, err := podcfg.GetJujuOCIImagePathFromModelRepo(modelImageRepo, vers)
 	if err != nil {
-		return nil, errors.Annotatef(err, "parsing %s", controller.CAASImageRepo)
+		return nil, errors.Trace(err)
 	}
-	imageRepo := params.NewDockerImageInfo(imageRepoDetails, registryPath)
-	logger.Tracef("imageRepo %v", imageRepo)
+
+	imageRepoDetails, err := docker.NewImageRepoDetails(modelImageRepo)
+	if err != nil {
+		return nil, errors.Annotatef(err, "parsing %s", modelImageRepo)
+	}
+
+	imageInfo := params.NewDockerImageInfo(imageRepoDetails, modelImagePath)
+
+	logger.Tracef("imageRepo %v", imageInfo)
```

Comment:
Remove this, you are logging credentials. :)

Reply:
yup you're right, not sure what was the background for adding this log, i have removed them in other places as well.